niworkflows/interfaces/freesurfer.py

```diff
@@ -372,9 +374,9 @@ def inject_skullstripped(subjects_dir, subject_id, skullstripped):
     if not op.exists(bm_auto):
         img = nb.load(t1)
         mask = nb.load(skullstripped)
-        bmask = new_img_like(mask, mask.get_data() > 0)
+        bmask = new_img_like(mask, mask.get_fdata() > 0)
         resampled_mask = resample_to_img(bmask, img, 'nearest')
-        masked_image = new_img_like(img, img.get_data() * resampled_mask.get_data())
+        masked_image = new_img_like(img, img.get_fdata() * resampled_mask.get_fdata())
```

Comment:
Since resampled_mask was created in memory, its dataobj is an `ndarray`. You could wrap it in `np.asanyarray()` if you want, which would be safe, but the following will be fine:

```suggestion
        masked_image = new_img_like(img, img.get_fdata() * resampled_mask.dataobj)
```